runtime/commonTest/src/kotlinx/serialization/json/JsonTestBase.kt

```diff
@@ -135,7 +134,7 @@ abstract class JsonTestBase {
         serializer: KSerializer<T>,
         data: T,
         expected: String,
-        json: Json = unquoted
+        json: Json = Json.plain
```

Comment:
Maybe just `Json(.Companion)`? Since you're going to deprecate plain anyway

Reply:
I'll merge with it separately